tokio/src/runtime/builder.rs

```diff
@@ -65,6 +66,9 @@ pub struct Builder {
 
     /// To run before each worker thread stops
     pub(super) before_stop: Option<Callback>,
+
+    /// Customizable keep alive timeout for BlockingPool
+    pub(super) keep_alive: Option<Duration>,
```

Comment:
Never mind, I was wrong when I said we didn't need to feature flag this — it's now dead code if the `blocking` feature is disabled, which [breaks the build](https://github.com/tokio-rs/tokio/pull/2809/checks?check_run_id=1065872719#step:8:14). We should be feature flagging this after all.

Sorry for giving you bad advice!

Reply:
I've reverted the last two commits. No problem at all.
I actually had in mind that there was a reason for which I had to go all-in with the features, but couldn't reproduce it right away in order to back my point.

Reply:
Yup, I think this is right. Sorry about that!